src/Concordium/Client/Runner.hs

```diff
@@ -3498,31 +3605,31 @@ processLegacyCmd action backend =
     GetConsensusInfo ->
       withClient backend $
         getConsensusInfoV2 >>=
-        printResponseValueAsJSON
+          printResponseValueAsJSON
     GetBlockInfo every block ->
       withClient backend $
         readBlockHashOrDefault Best block >>=
-        printBlockInfos every
+          printBlockInfos every
     GetBlockPendingUpdates block ->
       withClient backend $
         readBlockHashOrDefault Best block >>=
-        getBlockPendingUpdatesV2 >>=
-        printResponseValueAsJSON
+          getBlockPendingUpdatesV2 >>=
+            printResponseValueAsJSON
     GetBlockSpecialEvents block ->
       withClient backend $
         readBlockHashOrDefault Best block >>=
-        getBlockSpecialEventsV2 >>=
-        printResponseValueAsJSON
+          getBlockSpecialEventsV2 >>=
+            printResponseValueAsJSON
```

Comment:
Why were these changes necessary? Please avoid noise.

Reply:
They were not strictly, but I did not know how sequencing should be formatted, and this was consistent with how I ended up doing things elsewhere.

Reply:
Do I revert this now?

Reply:
fourmolu is not used in the client is it? I recall using it on a file in the client and it pretty much changed up everything.

Is what you are proposing that I use fourmolu config of concordium-base to see how it formats things (in this case sequencing) and then go from that that as a reference?